hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpCommandProcessor.java

```diff
@@ -102,6 +102,16 @@
     public static final String URI_LOG_LEVEL = "/hazelcast/rest/log-level";
     public static final String URI_LOG_LEVEL_RESET = "/hazelcast/rest/log-level/reset";
 
+    // Config
+    public static final String URI_CONFIG = "/hazelcast/rest/config";
+    public static final String URI_CONFIG_RELOAD = URI_CONFIG + "/reload";
+
+    // Internal APIs
+    public static final String URI_INTERNAL = "/hazelcast/rest/internal";
```

Comment:
as discussed on slack, this endpoint url is to be made public

Reply:
https://github.com/hazelcast/hazelcast/pull/20260/commits/ba36045516b542b20c07de897440549a55547fc2